src/main/java/au/gov/nla/dlir/models/Work.java

```diff
@@ -78,7 +78,8 @@
     private MarcData marcData;
     @JsonFormat(shape=JsonFormat.Shape.STRING, pattern=DATE_PATTERN)
     private Date expiryDate;
-
+    private String depositType;
+    private boolean hasOriginalCopy = false;
```

Comment:
Can this be renamed to something like `originalCopyAvailble`, just to follow boolean getter convention -`isOrginalCopyAvailable` rather than `getOriginalCopyAvailable`